### PR TITLE
STORM-1026: Expand external classpaths to include any jars

### PR DESCRIPTION
**Pull request by shellbj**

### CODE REVIEW

File: bin/storm.py [hunk 1]
```diff
@@ -103,6 +103,12 @@ def get_config_opts():
     sys.exit(1)
 
 def get_jars_full(adir):
+    files = []
+    if os.path.isdir(adir):
+        files = os.listdir(adir)
+    elif os.path.exists(adir):
+        files = [aidr]
+
     files = os.listdir(adir)
```
Comment (HeartSaVioR) [inline bin/storm.py]: This line should be removed to accomplish above lines.

Comment (shellbj) [inline bin/storm.py]: Good catch; updated.

**Comment by HeartSaVioR** [comment]
Sorry to being late, +1

**Comment by revans2** [comment]
@shellbj and @HeartSaVioR thanks for the patch and the review, sorry it took so long to get this merged in.

**Pull request merged**